tests/components/devolo_home_network/test_binary_sensor.py

```diff
@@ -0,0 +1,130 @@
+"""Tests for the devolo Home Network sensors."""
+from unittest.mock import AsyncMock, patch
+
+from devolo_plc_api.exceptions.device import DeviceUnavailable
+import pytest
+
+from homeassistant.components.binary_sensor import DOMAIN
+from homeassistant.components.devolo_home_network.const import (
+    CONNECTED_TO_ROUTER,
+    FIRMWARE_UPDATE_AVAILABLE,
+    LONG_UPDATE_INTERVAL,
+)
+from homeassistant.const import (
+    ENTITY_CATEGORY_DIAGNOSTIC,
+    STATE_OFF,
+    STATE_ON,
+    STATE_UNAVAILABLE,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry
+from homeassistant.util import dt
+
+from . import configure_integration
+from .const import PLCNET_ATTACHED
+
+from tests.common import async_fire_time_changed
+
+
+@pytest.mark.usefixtures("mock_device")
+@pytest.mark.usefixtures("mock_zeroconf")
```

Comment:
Not sure what the style recommendations are on this one so this is more of a note for core reviewers.
```suggestion
@pytest.mark.usefixtures("mock_device","mock_zeroconf")
```

Reply:
Anyway +1 for using `@pytest.mark.usefixtures`
With `VSCode` showing the arguments as faded to show they are unused, I find this explicit description much clearer.